controllers/error-handler.js

```diff
@@ -0,0 +1,9 @@
+function customErrors(err, req, res, next) {
+  if (err.status && err.msg) {
+    res.ststus(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+}
+
+module.exports = { customErrors };
```

Comment:
There's no need for this error handler right now. Please remove it until you need it. Otherwise you will find that your custom error tests will pass by default!